src/views/MyRequests/Table.tsx

```diff
@@ -17,6 +17,8 @@ import { formatDate } from 'utils/date';
 
 const Table = () => {
   const { t } = useTranslation('home');
+  const { t: tGovernance } = useTranslation('governanceReviewTeam');
```

Comment:
Can we load multiple namespaces like [this](https://react.i18next.com/latest/usetranslation-hook#loading-namespaces)? It's what the documentation recommends.